src/RpcNep5Tracker/RpcNep5Tracker.cs

```diff
@@ -108,33 +109,38 @@ private void RecordTransferHistory(StoreView snapshot, UInt160 scriptHash, UInt1
             if (eventName != "Transfer") return;
             if (stateItems.Count < 4) return;
 
-            if (!(stateItems[1] is null) && !(stateItems[1] is VM.Types.ByteArray))
+            if (!(stateItems[1].IsNull) && !(stateItems[1] is VM.Types.ByteArray))
                 return;
-            if (!(stateItems[2] is null) && !(stateItems[2] is VM.Types.ByteArray))
+            if (!(stateItems[2].IsNull) && !(stateItems[2] is VM.Types.ByteArray))
                 return;
             var amountItem = stateItems[3];
             if (!(amountItem is VM.Types.ByteArray || amountItem is VM.Types.Integer))
                 return;
-            byte[] fromBytes = stateItems[1]?.GetSpan().ToArray();
-            if (fromBytes?.Length != 20) fromBytes = null;
-            byte[] toBytes = stateItems[2]?.GetSpan().ToArray();
-            if (toBytes?.Length != 20) toBytes = null;
+            byte[] fromBytes = stateItems[1].IsNull ? null : stateItems[1].GetSpan().ToArray();
+            if (fromBytes?.Length != UInt160.Length) fromBytes = null;
+            byte[] toBytes = stateItems[2].IsNull ? null : stateItems[2].GetSpan().ToArray();
+            if (toBytes?.Length != UInt160.Length) toBytes = null;
             if (fromBytes == null && toBytes == null) return;
```

Comment:
If someone send `new (byte[50],new byte[20])` it will be treated as `(null,new byte[20])` We should distinguish in L120 between a format error and `null`.

Reply:
so maybe we should just `return` if the address format is invalid?

Reply:
I think so